app/controllers/precheck/confirmation_controller.rb

```diff
@@ -2,6 +2,9 @@ class Precheck::ConfirmationController < PrecheckController
   def create
     redirect_to precheck_status_path and return unless Precheck::EligibilityService.new(family: @family).call
 
+    @hotel = params['hotel'] === 'Other' ? " OTHER - #{params['other_hotel']} " : params['hotel']
```

Comment:
Is the triple-equal really needed here? Is `@hotel` needed by the view?

Reply:
Sorry I just had the stomach flu Thursday/Friday, so not thinking straight.  Good catch, hotel is not passed to the view here so will put as a local variable.  And no, its not necessary for the case quality operator... I was coding in JS/PHP dynamic language at the same time and it bled into Ruby.  Will correct these.